bin/ci

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "../loader"
+
+if Config.ci_hetzner_hostname.nil?
+  fail "Set CI_HETZNER_HOSTNAME env variable"
+end
+
+if Config.ci_hetzner_server_id.nil?
+  fail "Set CI_HETZNER_SERVER_ID env variable"
+end
+
+st = Prog::Test::HetznerServer.assemble(Config.ci_hetzner_hostname, Config.ci_hetzner_server_id)
```

Comment:
Is there a way not add these two configuration items? I find it odd that any special CI configuration is required.

Reply:
I guess a particular VmHost has to be sacrificed to be reset continuously? is that an interesting constraint?